tools/pepsi/src/pre_game.rs

```diff
@@ -47,6 +47,7 @@ pub struct Arguments {
     /// The NAOs to upload to with player number assignments e.g. 20w:2 or 10.1.24.22:5 (player numbers start from 1)
     #[arg(required = true)]
     pub assignments: Vec<NaoAddressPlayerAssignment>,
+    pub skip_has_stable_os_version: bool,
```

Comment:
You changed this flag's name. Could we improve that you currently have two verbs after another in the beginning? Maybe "skip stable os version check" or so...

Reply:
I would drop the "stable" and perhaps "version" too.
`skip-os-check` or `skip-os-version-check` should be sufficient.

"stable" only describes one of the use cases for this check, we want to avoid uploading to a NAO with an outdated OS image as well as one that has an newer one.
The latter case may be because the branch we compiled isn't based on the latest main branch or because the robot has an experimental image. "stable" only applies to this last, experimental case IMO.